themes/classic/assets/css/theme.css

```diff
@@ -4823,7 +4818,11 @@ ul {
     margin-top: 1rem; }
   section.checkout-step {
     width: 100%; }
-  section.checkout-step input[name=address1], section.checkout-step input[name=address2], section.checkout-step input[type=text], section.checkout-step input[type=password], input[type=date] {
+  section.checkout-step input[name=address1],
+  section.checkout-step input[name=address2],
+  section.checkout-step input[type=text],
+  section.checkout-step input[type=password],
```

Comment:
this is completely not flexible, hardcoded fields? what about modules which will modify these?